mcs/class/System/Mono.UnityTls/UnityTlsConversions.cs

```diff
@@ -23,7 +23,7 @@ public static UnityTls.unitytls_protocol GetMinProtocol (SslProtocols protocols)
 				return UnityTls.unitytls_protocol.UNITYTLS_PROTOCOL_TLS_1_1;
 			if (protocols.HasFlag (SslProtocols.Tls12))
 				return UnityTls.unitytls_protocol.UNITYTLS_PROTOCOL_TLS_1_2;
-			return UnityTls.unitytls_protocol.UNITYTLS_PROTOCOL_TLS_1_2;	// Behavior as in AppleTlsContext
+			return UnityTls.unitytls_protocol.UNITYTLS_PROTOCOL_TLS_1_0;	// Use mbed tls min default
```

Comment:
So this is a change in default behavior? Should we use 1.2 here still?

Reply:
In the past the value passed here was `Default = 0xF0`.  This would cause these routines to return Min and Max as `UNITYTLS_PROTOCOL_TLS_1_0`  However maybe we should consider setting Min and Max to 1.2 as 1.1 and 1.0 are now obsolete in industry.

Reply:
Tempting, but I figure that could break plenty of users that don't pay as much attention what is secure and what not.

Don't understand the mbed tls comment here - UnityTls abstracts over mbedtls and we had other backends in the past (and will have again as GamesCore is moving to a different one).

Reply:
I found that we were an abstraction over mbedtls. I looked up the the min and max versions for the online documentation as my guide.